pkg/probe/log_fortianalyzer.go

```diff
@@ -0,0 +1,55 @@
+package probe
+
+import (
+	"log"
+
+	"github.com/bluecmd/fortigate_exporter/pkg/http"
+	"github.com/prometheus/client_golang/prometheus"
+)
+
+type LogAnaResults struct {
+	Registration string `json:"registration"`
+	Connection   string `json:"connection"`
+	Received     int    `json:"received"`
+}
+
+type LogAna struct {
+	Results LogAnaResults `json:"results"`
+	VDOM    string        `json:"vdom"`
+	Version string        `json:"version"`
+}
+
+func probeLogAnalyzer(c http.FortiHTTP, meta *TargetMetadata) ([]prometheus.Metric, bool) {
+	var (
+		logAnaReg = prometheus.NewDesc(
+			"fortigate_log_fortianalyzer_registration",
+			"Fortianalyzer registration state",
+			[]string{"vdom", "registration"}, nil,
+		)
+		logAnaCon = prometheus.NewDesc(
+			"fortigate_log_fortianalyzer_connection",
+			"Fortianalyzer connection state",
+			[]string{"vdom", "connection"}, nil,
+		)
+		logAnaRcv = prometheus.NewDesc(
+			"fortigate_log_fortianalyzer_received",
```

Comment:
```suggestion
			"fortigate_log_fortianalyzer_logs_received",
```
IMO this is more readable